.unit-test/human-logic-test.js

```diff
@@ -0,0 +1,34 @@
+(function () {
+    'use strict';
+    describe('Test CharacterLogic', function () {
+        var characterLogic,
+            coreData,
+            sandbox;
+
+        beforeEach(function(){
+            module('Services.MyModule');
+            module(function($provide){
+                $provide.service('CharacterLogic', mocks.CharacterLogic);
+                $provide.service('CoreData', mocks.CoreData);
+            });
+            sandbox = sinon.sandbox.create();
+            characterLogic = sinon.sandbox.mock(mocks.CharacterLogic);
```

Comment:
Mock id probably the wrong type of observer - you probably want a spy or stub. As there is other stuff to fix - fix this too.